renderer/src/components/ChartsDashboard.jsx

```
@@ -0,0 +1,251 @@
+      </div>
+
+      {/* Map Tools Charts Section (conditional) */}
+      {features.mapToolsEnabled && (
+        <div className="mb-12">
+          <div className="mb-6 flex justify-between items-center">
+            <h2 className="text-2xl font-semibold text-gray-800">Character Activity Charts</h2>
+            <button 
+              onClick={sendAllActivityCharts}
+              disabled={sendingAllCharts}
+              className="px-4 py-2 bg-indigo-600 text-white rounded-md hover:bg-indigo-700 transition-colors flex items-center space-x-2 disabled:opacity-50"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)